npm-shrinkwrap.json

```diff
@@ -936,6 +948,26 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
       "dev": true
     },
+    "fury-adapter-apib-parser": {
```

Comment:
This was already in `package.json` but for some reason missing from the shrinkwrap. Regenerated shrink wrap and it came back.